Project.toml

```diff
@@ -0,0 +1,17 @@
+name = "DiffResults"
+uuid = "163ba53b-c6d8-5494-b064-1a9d43ac40c5"
+version = "1.0.0"
+
+[deps]
+StaticArrays = "90137ffa-7385-5640-81b9-e52037218182"
+
+[compat]
+StaticArrays = "0.8"
+julia = "1"
+
+[extras]
+DiffResults = "163ba53b-c6d8-5494-b064-1a9d43ac40c5"
```

Comment:
I don't think packages need to declare themselves as their own dependency for testing :thinking: